test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -126,5 +253,48 @@ private void Grid_KeyDown(object sender, KeyRoutedEventArgs e)
                 Frame.GoBack();
             }
         }
+
+        private void Sv_Tapped(object sender, TappedRoutedEventArgs e)
+        {
+            if (!isScrolling)
```

Comment:
Is it possible to tap an item while panning ? would that not stop the pan ?

Reply:
Tapping on an item while panning will interrupt panning and select whatever item was closest to center when scrolling stopped. This is by design since it's unrealistic to assume that the user made a legitimate selection while the scrolling was spinning, since the user wouldnt' be able to know what they were tapping.